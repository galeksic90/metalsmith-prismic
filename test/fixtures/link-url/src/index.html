---
template: index.hbt
prismic:
  macaron:
    query: '[[:d = at(document.tags, ["Macaron"])] [:d = any(document.type, ["product"])]]'
---
