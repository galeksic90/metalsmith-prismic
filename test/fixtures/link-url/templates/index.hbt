<html>
<body>
<ul>
{{#each prismic.macaron.results}}
    <li style="background-image: url({{{data.image.url}}});">
        {{{data.name.html}}}
        <a href="{{data.related.url}}">Related</a>
    </li>
{{/each}}
</ul>
</body>
</html>
