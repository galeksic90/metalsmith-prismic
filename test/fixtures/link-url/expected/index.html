<html>
<body>
<ul>
    <li style="background-image: url(https://prismic-io.s3.amazonaws.com/lesbonneschoses/79a7a068ceb47892ac8de2480e878962d8b93cc6.png);">
        <h1>Cool Coconut Macaron</h1>
        <a href="/product/UlfoxUnM0wkXYXbH/vanilla-macaron">Related</a>
    </li>
    <li style="background-image: url(https://prismic-io.s3.amazonaws.com/lesbonneschoses/604400b41b2e275ee766bd69b69b33734043aa38.png);">
        <h1>Salted Caramel Macaron</h1>
        <a href="/product/UlfoxUnM0wkXYXbE/dark-chocolate-macaron">Related</a>
    </li>
    <li style="background-image: url(https://prismic-io.s3.amazonaws.com/lesbonneschoses/78ef921308fe727a88530a76f6ab6e1123cdd147.png);">
        <h1>Speculoos Macaron</h1>
        <a href="/product/UlfoxUnM0wkXYXbT/speculoos-cupcake">Related</a>
    </li>
    <li style="background-image: url(https://prismic-io.s3.amazonaws.com/lesbonneschoses/acc69618fab22be90aed95b39b91b62a80595a6b.png);">
        <h1>Black &amp; White Macaron</h1>
        <a href="/product/UlfoxUnM0wkXYXbD/speculoos-macaron">Related</a>
    </li>
    <li style="background-image: url(https://prismic-io.s3.amazonaws.com/lesbonneschoses/b6d184431d8f2902b97983e6469ffc577da751b3.png);">
        <h1>Pistachio Macaron</h1>
        <a href="/product/UlfoxUnM0wkXYXbH/vanilla-macaron">Related</a>
    </li>
    <li style="background-image: url(https://prismic-io.s3.amazonaws.com/lesbonneschoses/90954eac02861c4134431e3acb4f27938c5c5ce5.png);">
        <h1>Dark Chocolate Macaron</h1>
        <a href="/product/UlfoxUnM0wkXYXbe/woodland-cherry-pie">Related</a>
    </li>
    <li style="background-image: url(https://prismic-io.s3.amazonaws.com/lesbonneschoses/fe939eeb67713903f7f68c31efc98e52e1c7655d.png);">
        <h1>Vanilla Macaron</h1>
        <a href="/product/UlfoxUnM0wkXYXbE/dark-chocolate-macaron">Related</a>
    </li>
</ul>
</body>
</html>
